### PR TITLE
Companion single Color patch.

### PR DESCRIPTION
**Pull request by Adenilson**

### CODE REVIEW

**Comment by Adenilson** [comment]
Rebased with current master, executed ref-tests (OSX Yosemite), everything looks fine.

File: components/gfx/color.rs [hunk 1]
```diff
@@ -3,9 +3,12 @@
...
 
 use azure::AzFloat;
-use azure::azure_hl::Color as AzColor;
+use azure::azure::AzColor;
 
-pub type Color = AzColor;
+#[inline]
+pub fn new(r: AzFloat, g: AzFloat, b: AzFloat, a: AzFloat) -> AzColor {
+         AzColor { r: r, g: g, b: b, a: a }
+    }
```
Comment (pcwalton) [inline components/gfx/color.rs]: nit: weird indentation here

Comment (pcwalton) [inline components/gfx/color.rs]: Also, why does this need to exist at all? Shouldn't this be moved into `rust-azure` if anything?

Comment (pcwalton) [inline components/gfx/color.rs]: After discussion on IRC this is fine to leave here. But please fix the indentation :)

**Comment by Adenilson** [comment]
Updated.

**Comment by pcwalton** [comment]
Squash and ready to merge!

**Pull request merged**